pkg/yurtctl/util/edgenode/common.go

```diff
@@ -60,29 +60,34 @@ metadata:
   namespace: kube-system
 spec:
   volumes:
-  - name: hub-dir
+  - name: pki
     hostPath:
-      path: /var/lib/yurthub
-      type: DirectoryOrCreate
+      path: /etc/kubernetes/pki
+      type: Directory
```

Comment:
`/var/lib/yurthub` should be kept for yurthub certificates storage. and `/etc/kubernetes/pki` is not need to added because `/etc/kubernetes` hostpath has already mounted.